packages/node-image-non-compute-common/base.packages

```diff
@@ -7,6 +7,7 @@ amsd=2.4.1-1571.4.sles15
 apparmor-profiles=2.13.6-150300.3.11.2
 arptables=0.0.4-8.39
 at=3.1.20-11.3
+aws-cli=1.20.7-30.3.1
```

Comment:
Should we add this to the LiveCD as well?

Reply:
It may be useful to NCNs booting without a publicly readable S3 bucket. In that case it could be used in the LiveCD I suppose. For UANs, it won't be needed until the full CSM stack is running.

Reply:
@rustydb would you like me to add it to the LiveCD?

Reply:
yes plz

Reply:
I added it.